recipes/debug_assert/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+cmake_minimum_required(VERSION 2.8.12)
+project(test_package)
+
+set(CMAKE_VERBOSE_MAKEFILE TRUE)
+set(CMAKE_CXX_STANDARD 11)
```

Comment:
```suggestion
project(test_package)
```

Reply:
I will leave the CMAKE_CXX_STANDARD since conan handling of this is still a bit broken (Experimental :P)